src/main/java/io/appform/dropwizard/actors/connectivity/RMQConnection.java

```diff
@@ -94,6 +94,7 @@ public void start() throws Exception {
         factory.setTopologyRecoveryEnabled(true);
         factory.setNetworkRecoveryInterval(3000);
         factory.setRequestedHeartbeat(60);
+        factory.setVirtualHost(config.getVirtualHost());
```

Comment:
Check null or empty before setting. If it is null or empty, do not set it. Factory takes a default value of `/`

Reply:
Added null and empty checks